src/main/java/edu/uc/inventorymanager/ui/MainController.java

```diff
@@ -40,7 +40,6 @@ public String index(Model model) {
 
     @RequestMapping("/saveItem")
     public String saveItem(Item item) {
-        //itemService.save(item);
```

Comment:
Removed commented out code to improve quality of code. This helps code be more readable and easier to maintain, reducing technical debt. Source: https://martinfowler.com/bliki/TechnicalDebt.html